test/runtests.jl

```diff
@@ -2,179 +2,122 @@
-    smb_spk("b", "DES= 2099942;", "2021-1-1", "2029-4-13T21:46:07.999", "joe@your.domain.name", "2099942_.bsp", ftp_verbose=true)
-    @test isfile("2099942_.bsp")
+    rm(local_file)
 end
 
 @testset "Generation and file download of small-bodies binary SPK files: smb_spk_ele" begin
-    elements_str = "EPOCH=2449526.5 EC=.6570220840219289 QR=.5559654280797371 TP=2449448.890787227 OM=78.10766874391773 W=77.40198125423228 IN=24.4225258251465"
-    smb_spk_ele("b", "1990 MU", DateTime(2021,Jan,1), DateTime(2022,Jan,1), elements_str, "joe@your.domain.name", "1990mu.bsp")
-    @test isfile("1990mu.bsp")
-    elements_str = "EPOCH= 2454733.5 EC=.1911952942528226 QR=.7460724385331012 TP=2454894.912507658200 OM=204.4460284242489 W=126.401880836064 IN=3.331369228495799"
-    ftp_name, local_file = smb_spk_ele("b", "99942", DateTime(2021,Jan,1), DateTime(2022,Jan,1), elements_str, "joe@your.domain.name")
-    @test isfile(local_file)
+    # Generate a binary SPK file for asteroid 99942 Apophis from its osculating elements
     epoch = 2454733.5
     ec = 0.1911952942528226
     qr = 0.7460724385331012
     tp = 2454894.912507658200
     om = 204.4460284242489
     w = 126.401880836064
     inc = 3.331369228495799
-    ftp_name, local_file = smb_spk_ele("b", "99942", DateTime(2021,Jan,1), DateTime(2022,Jan,1), epoch, ec, qr, tp, om, w, inc, "joe@your.domain.name")
+    local_file = smb_spk_ele("99942", DateTime(2021, Jan, 1), DateTime(2022, Jan, 1),
+                             epoch, ec, qr, tp, om, w, inc)
     @test isfile(local_file)
-    ftp_name, local_file = smb_spk_ele("b", "99942", DateTime(2021,Jan,1), DateTime(2022,Jan,1), elements_str, "joe@your.domain.name", ftp_verbose=true)
+    rm(local_file)
+    
+    # Generate a binary SPK file for asteroid 1990 MU from its osculating elements
+    epoch = 2449526.5
+    ec = 0.6570220840219289
+    qr = 0.5559654280797371
+    tp = 2449448.890787227
+    om = 78.10766874391773
+    w = 77.40198125423228
+    inc = 24.4225258251465
+    local_file = smb_spk_ele("1990 MU", DateTime(2021, Jan, 1), DateTime(2022, Jan, 1),
+                             epoch, ec, qr, tp, om, w, inc)
+    @test isfile(local_file) 
+    rm(local_file)
+end
+
+@testset "Vector table generation: vec_tbl" begin
+    # Generate tables and save output to Apophis.txt
+    t_start = DateTime(2029, 4, 13)
+    t_stop = Date(2029, 4, 14)
+    δt = Hour(1)
+    local_file = vec_tbl("Apophis;", t_start, t_stop, δt; FILENAME = "Apophis.txt", CENTER = "@ssb",
+                         REF_PLANE = "FRAME", OUT_UNITS = "AU-D", CSV_FORMAT = true, VEC_TABLE = 2)
+
     @test isfile(local_file)
+
+    apophisvt = vec_tbl("Apophis;", t_start, t_stop, δt; CENTER = "@ssb", REF_PLANE = "FRAME",
+                        OUT_UNITS = "AU-D", CSV_FORMAT = true, VEC_TABLE = 2)
+
+    x = readlines(local_file)
+    y = split(apophisvt, "\n")[1:end-1]
+
+    @test x == y
```

Comment:
Tests are failing in 1.9 and 1.10, I'd suggest to change this test to something like this:
```suggestion
    @test length(x) == length(y)
    lx = length(x)
    inds = setdiff(1:lx,[2,35]) # avoid lines 2 and 35, since they have time of retrieval
    @test x[inds] == y[inds]
```

Reply:
(edit: above I meant julia 1.6; wrote 1.9 instead)